packages/ui-components/src/lib/inputs/InputSelect/InputSelect.tsx

```diff
@@ -298,9 +300,10 @@ export const InputSelect: FC<InputSelectProps> = ({
   // Allows the input to keep it's value when the user selects an option when disableCloseOnSelect is true
   if (isMultiselect && autocompleteProps.disableCloseOnSelect) {
     autocompleteProps.inputValue = inputValue;
+    const existingOnBlur = autocompleteProps.onBlur;
     autocompleteProps.onBlur = event => {
-      setInputValue('');
-      if (autocompleteProps.onBlur) autocompleteProps.onBlur(event);
```

Comment:
this was causing an infinite recursive function. We need to pun the function as a const before overwriting it.